tools/collect_metrics.py

```diff
@@ -46,6 +46,18 @@ def load_pickle(path: str):
     ),
 }
 
+model_link_mappings = {
+    "BERT": "tests/models/bert",
+    "Bloom": "tests/models/bloom",
+    "Falcon": "tests/models/falcon",
+    "GPT-2": "tests/models/gpt2",
+    "Llama": "tests/models/llama",
+    "Mnist (Eval)": "tests/models/mnist",
+    "Mnist (Train)": "tests/models/mnist",
+    "ResNet18": "tests/models/resnet",
+    "YOLOS": "tests/models/yolos",
```

Comment:
I'd love us to get rid of this over time. It should be enough to just add a test, there should be no need to "register it"

Reply:
Yeah this could be reworked. My rational is that sometimes the model name and associated directory are different, so I used a mapping to link them. But we could also have this data generated from the test model itself as another way.